APE10.rst

```diff
@@ -0,0 +1,155 @@
+Roadmap for Python 3-only support
+---------------------------------
+
+author: Tom Robitaille
+
+date-created: 2016 July 13
+
+date-last-revised: 2016 July 13
+
+type: Process
+
+status: Draft
+
+Abstract
+--------
+
+The purpose of this APE is to set out a roadmap for future releases of the
+Astropy core package, in particular to identify the process for switching to
+releases that support only Python 3.
+
+Detailed description
+--------------------
+
+A number of scientific Python projects, including IPython, Matplotlib, Sympy,
+and others have now signed the `Python 3 statement
+<https://python3statement.github.io>`_ which states that scientific Python
+packages will drop support for Python 2 no longer than 2020. In fact, the
```

Comment:
Also on this line: maybe "In fact" -> "already" or something like that? ("in fact" implies sort of surprise, and we want to make this *less* surprising)